soc/nxp/imx/imx95/Kconfig.defconfig.mimx95.m7

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_MIMX9596_M7
+
+config FLASH_SIZE
+	default $(dt_node_reg_size_int,/soc/itcm@0,0,K)
```

Comment:
nitpick: we can use `zephyr,flash` here which references `itcm`. Same applies to `FLASH_BASE_ADDRESS`.

Reply:
Okay.